mne_sandbox/preprocessing/_sns.py

```diff
@@ -64,6 +73,11 @@ def fit(self, raw, verbose=None):
         -------
         sns : Instance of SensorNoiseSuppression
             The modified instance.
+
+        Notes
+        -----
+        In the resulting operator, bad channels will be reconstructed by
+        using the good channels.
```

Comment:
why does SNS need to know bad channels? I think you mentioned it somewhere but I can't find the comment any more.